.github/workflows/python-oda-api.yml

```diff
@@ -56,5 +56,5 @@ jobs:
     
     - name: Test oda_api with pytest
       run: |
-        cd oda_api && DISPATCHER_MOCK_KB=yes python -m pytest tests -sv --full-trace --log-cli-level=DEBUG 
+        cd oda_api && DISPATCHER_MOCK_KB=yes python -m pytest tests -m 'not slow and not test_drupal' -sv --full-trace --log-cli-level=DEBUG 
```

Comment:
this is already in master, right?

Reply:
no it is not

Reply:
Oh! Where is it then? Did it not come from gallery pr?

Reply:
I just added the `-m 'not slow and not test_drupal'` option, and that was not there when inserted this the first time.

Reply:
Whereas the python-oda-api workflow is already in the master, I just added an option to skip certain tests

Reply:
test_drupal is testing gallery, right?

Reply:
exactly

Reply:
why was it not excluded in gallery PR then?

Reply:
because the gallery related functionalities and tests inside oda_api were introduced later

Reply:
but how did the test pass in workflows when the tests in oda_api were introduced?

Reply:
probably the drupal gallery tests were not included yet